.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  # Node
+  - package-ecosystem: "npm"
```

Comment:
Shouldn't this be `pnpm` ?

Reply:
I even included the link in the PR description! :sigh: No, for all three, `npm`, `yarn` and `pnpm`, the `package-ecosystem` value should be `npm`.

Reply:
OOH, no 😂